### DISCUSSION TITLE
Add "ENTER" to submit and open new entry on shopping lists

### DISCUSSION THREAD

**Discussion by StarkZarn** [mealie-recipes/mealie#2686] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Currently shopping list interaction requires mouse movement to click the save button for each and every entry. A keyboard-only workflow as an option would be much smoother.

### Please provide a concise description of the feature that would resolve your issue.

Quality of life - "enter"/"return" opens a new a line in the text field in shopping lists. In my opinion it should submit the current text and open a new line, so multiple entries can be added with only keyboard interaction.

### Please consider and list out some caveats or tradeoffs made in your design decision

Users who may depend on CRLF within a single shopping list entry would have to adjust to a shift+click workflow. This seems unlikely to be a majority of people though.

Additionally, a good example of how this might work would be the lists feature in Planka, if anyone is unsure about what I'm getting at.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by legrego** [comment]
Related: https://github.com/mealie-recipes/mealie/discussions/2683

**Reply by StarkZarn** [reply]
Cheers. Apologies for not finding this in my cursory search prior to opening.
